### PR TITLE
Fix indent using Visual Code Formatter 

### PR DESCRIPTION
**Pull request by alagoutte**
i use 4 spaces indent

add also add .vscode settings

### CODE REVIEW

**Comment by alagoutte** [comment]
@LifeOfBrianOC and @GaryJBlake can you look ?

File: powerVCF.psd1 [hunk 1]
```diff
@@ -8,41 +8,41 @@
 
 @{
 
-# Script module or binary module file associated with this manifest.
-RootModule = 'PowerVCF.psm1'
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'PowerVCF.psm1'
```
Comment (GaryJBlake) [inline powerVCF.psd1:12]: Sorry what value are you trying to add here, the file was auto generated by the PowerShell tools

Comment (alagoutte) [reply powerVCF.psd1:12]: Yes, i known but we don't regenerate after...

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Place your settings in this file to overwrite default and user settings.
```
[the remark this answers is not in the document]
Comment (alagoutte) [reply .vscode/settings.json:1]: Yes but it is better to have the same indent on the file (and it will be possible to use tools for fix/force indent...)

**Comment by alagoutte** [comment]
rebase my patch with 2.0.0 (and remove psd1 change)

**Comment by alagoutte** [comment]
Hi Guys,

There is a conflict but what the better solution for hope to be merged ? 

Because without fixed indent, Visual Code is crazy...

**Comment by alagoutte** [comment]
Hello,

Update the change to only change indent on psm1 file

**Comment by alagoutte** [comment]
Hi Guys @GaryJBlake @LifeOfBrianOC 

Can you look ? i have rebase the patch and only modified psm1

**Comment by alagoutte** [comment]
Thanks !!

**Pull request merged**